plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/JfrogArtifactoryRepository.tsx

```diff
@@ -22,15 +22,17 @@ const useLocalStyles = makeStyles({
   },
 });
 
-export function JfrogArtifactoryRepository(props: RepositoryProps) {
+export function JfrogArtifactoryRepository(
+  props: RepositoryProps = { title: 'Docker Images' },
+) {
   const jfrogArtifactoryClient = useApi(jfrogArtifactoryApiRef);
   const classes = useStyles();
   const localClasses = useLocalStyles();
   const [edges, setEdges] = useState<Edge[]>([]);
   const title = `Jfrog Artifactory repository: ${props.image}`;
 
   const { loading } = useAsync(async () => {
-    const tagsResponse = await jfrogArtifactoryClient.getTags(props.image);
+    const tagsResponse = await jfrogArtifactoryClient.getTags(props.image!);
```

Comment:
this may not be needed.

Reply:
If i don't give the `!` then some tests are getting failed due to some undefined string value.

Reply:
with the above change that might not be needed, i think as it will be the same as before and you only pass the default value to the title